patches/net/minecraft/world/item/BlockItem.java.patch

```diff
@@ -31,16 +31,25 @@
      @Nullable
      public BlockPlaceContext updatePlacementContext(BlockPlaceContext p_40609_) {
          return p_40609_;
-@@ -193,6 +_,12 @@
- 
-     public void registerBlocks(Map<Block, Item> p_40607_, Item p_40608_) {
+@@ -195,6 +_,10 @@
          p_40607_.put(this.getBlock(), p_40608_);
-+    }
-+
-+    /** @deprecated Neo: To be removed without replacement since registry replacement is not a feature anymore. */
-+    @Deprecated(forRemoval = true, since = "1.21.1")
```

Comment:
This deprecation should not be reverted

Reply:
Weird im not seeing this deprecation in my code base, why is it in the patches?

![image](https://github.com/user-attachments/assets/91be3b32-eb40-44f3-82dc-02f47c80c406)

Reply:
Annotation should be restored in latest commit